content/source/docs/cloud/workspaces/agent.html.md

```diff
@@ -0,0 +1,175 @@
+1. Verify the integrity of the downloaded archive, as well as the signature of the `SHA256SUMS` file using the instructions available on [HashiCorp's security page](https://www.hashicorp.com/security).
+1. Extract the release archive. The `unzip` utility is available on most Linux distributions and may be invoked as `unzip <archive file>`. Two individual binaries will be extracted (`tfc-agent` and `tfc-agent-core`). These binaries _must_ reside in the same directory for the agent to function properly.
+
+#### Start the Agent
+
+Using the Agent token you copied earlier, set the `TFC_AGENT_TOKEN` and `TFC_AGENT_NAME` environment variables.
+
+```
+export TFC_AGENT_TOKEN=your-token
+export TFC_AGENT_NAME=your-agent-name
+./tfc-agent
+```
+
+-> **Note:** The `TFC_AGENT_NAME` variable is optional. If you do not specify a name here, one will not be displayed. These names are for your reference only, and the agent ID is what will appear in logs and API requests.
+
+Once complete, your agent will appear on the Agents page and display its current status.
+
+![Screenshot: The Agents page with one connected agent in the idle state](./images/agent-running.png)
+
+##### Optional Configuration: Running an Agent using Docker
+
+Alternatively, you can use our official agent Docker container to run the Agent.
+
+```
+docker pull hashicorp/tfc-agent:latest
+docker run -e TFC_AGENT_TOKEN=your-token -e
+TFC_AGENT_NAME=your-agent-name hashicorp/tfc-agent
+```
+
+##### Optional Configuration: Single-execution mode
+
+The Agent can also be configured to run in single-execution mode, which will ensure that the Agent only runs a single workload, then terminates. This can be used in combination with Docker and a process supervisor to ensure a clean working environment for every Terraform run.
+
+To use single-execution mode, start the agent with the `-single` command line argument.
+
+### Configuring Workspaces to use the Agent
+
+-> **Note:** You must have “Admin” access to the workspace you are configuring to change the [execution mode](/docs/cloud/workspaces/settings.html#execution-mode). ([More about permissions.](/docs/cloud/users-teams-organizations/permissions.html))
+
+[permissions-citation]: #intentionally-unused---keep-for-maintainers
+
+~> **Important:** Changing your workspace's [execution mode](/docs/cloud/workspaces/settings.html#execution-mode) after a run has already been planned will cause the run to error when it is applied.
+To minimize the number runs that error, you should disable [auto-apply](/docs/cloud/workspaces/settings.html#auto-apply-and-manual-apply),
+complete any runs that are no longer in the [pending stage](/docs/cloud/run/states.html#1-the-pending-stage),
+and [lock](/docs/cloud/workspaces/settings.html#locking) your workspace before changing the execution mode.
+
+To configure a workspace to execute runs using an agent:
+
+1. Open the workspace from the main Workspaces view, then navigate to Settings > General from the dropdown menu.
```

Comment:
```suggestion
1. Open the workspace from the main **Workspaces** view, then navigate to **Settings > General** from the dropdown menu.
```

Reply:
Ah, good catch. The convention we've been using in Education (and elsewhere in the TFC docs) is to quote UI text instead of bolding it, so I'll go with that. 

```suggestion
1. Open the workspace from the main "Workspaces" view, then navigate to "Settings > General" from the dropdown menu.
```